VirtoCommerce.InventoryModule.Web/Scripts/blades/fulfillment-center-detail.js

```diff
@@ -40,10 +40,12 @@ angular.module('virtoCommerce.inventoryModule')
 
         function saveChanges() {
             blade.isLoading = true;
-            fulfillments.update({}, blade.currentEntity, function (center) {
-                blade.currentEntityId = center.id;
-                blade.refresh(true);
+            fulfillments.update(blade.currentEntity, function (centers) {
+                if (centers.length) {
```

Comment:
Do we need changes here if we reverted API? Probably you could revert this file.

Reply:
reverted